wsu-people-directory.php

```diff
@@ -14,8 +14,16 @@
 	die;
 }
 
-// Flush rewrite rules on activation or deactivation.
-register_activation_hook( __FILE__, 'flush_rewrite_rules' );
+
+register_activation_hook( __FILE__, 'wsuwp_people_directory_activate' );
```

Comment:
We may want to ditch the activation hook entirely for rewrite rules. It still doesn't fire right when a plugin is network activated, so something should exist that always checks the current state and matches it against expectations. In our case this can be an `upgrade_routine()` method (or similar) that fires on `init` or something.